### ISSUE TITLE
Casino tax

### ISSUE THREAD

**Issue by BigTony7** [HollywoodHeists/public#201]
Good evening to all, I understand this topic may be a bit conservational but I believe that the casino tax should be changed. Getting 35% of your chips is not fair, if I buy 5k worth of chips and win 10k and take out the money I get 3500 cash back, I lose 1500 which is pointless to even be at the casino, yes I have won a lot of chips but even if I take out the chips I lose so much which makes me not even want to take chips out. I truly believe if we were to receive a 75% to 70% tax on our chips it would be fair, more people would go enjoy the casino than to do a drive through for the wheel only. I truly hope you take this into consideration, and please understand that 75% is a reasonable amount. Anything under is not going to be worth going to the casino and actually enjoying the experience there.

**Comment by ChugThis** [comment]
As far as details of the casino I have next to none, but I do know that the tax is only so severe when there isn’t money in it. It’s because so many people win money, and then win money, and the daily wheel payouts that it keeps going under so much. I’ve been paid out in full before but only once and I never saw it go back again, even dumped chips in one day to see. I think at this point it’s probably already WAY in debt as a “business” and with how the economy works I don’t know if they are looking to effectively spawn money. I agree something should probably change (and while I know you’ve had some potentially great payouts) I think it’s also to avoid a grind mentality of the casino

**Comment by ChugThis** [comment]
I don’t know if a jump to 75% would be my first change, maybe 50? Big enough that big paydays still worth? Or possibly other incentives for people to actually stick around and bet? Or maybe it changes based on the game you’re playing? Slots don’t pay out as high as poker for example. Just some ideas

**Comment by BigTony7** [comment]
50% of your winnings still isn't much, there would be no point of trying to win big because you know you'll get only half of that, if someone were to win 100k and only get 50k out of that its not really worth it. Yes it's a high price but still won't get people to constantly to stay and enjoy it. No one stays to play blackjack, slots, roulette etc. 75% is a fair price. There isn't really any big winning games there, only the hores but it's very rare to get a win, I know because I only won 3 times out of the 100 plus times I played. Blackjack your max bet is 10k at the none high roller area, the max you can win in one hand is around 20k, only being able to get 10k back isn't really exciting. Even then most people don't take out much chips because they know they aren't getting any money back regardless. 75% may seem high but in reality people would be losing 25% of cash which is way better that 65% of their cash. It won't hurt the economy either, ive spoke to a couple people and the crim life doesn't have that much stuff either. But even then you need dirty for anything along those crim lines.

**Comment by ChugThis** [comment]
Yeah I suppose the idea of essentially spawning money from the casino when every other business is held to an actual economy is my main thought.  This is also beta, and I’m sure the casino isn’t complete. I’m sure there is something the team will come up with for this. Also, your idea of “worth it” doesn’t make much sense to me as I don’t think it’s intended to be a money making job essentially, unless you’re spending hours there or losing a ton of money. You’ve definitely been there more than I have so you have more of an idea of it, but also have to think about when there is potentially a city full of people going to the casino, which in theory would solve the issue anyway once money is being paid into the account. So many chips are being won and no cash being put in, that’s the root of the issue, but I’ll let the guys make the decisions and speak with you about it

**Comment by BigTony7** [comment]
when i mean worth it, i dont mean as in someone spending hours in there, i feel like you would lose more money than earning in there, to me a casino trip is just spinning the wheel and betting on the horses then leaving, i dont think  people would stay in there for hours because you end up losing more money than earning and its only a couple things you can do so it would get boring after a while. even if the player count is high, taking more cash away from the chips you earned or bought, no one would really want to even go to the casino it would be counter intuitive. why even bother going and playing cards when you know the possibility of wining is very hard and winging big is even harder and the reward is basically nothing or even less than what you put in. im just trying to get people to come and actually enjoy the casino, if the casino can add more things (im not sure if they can) it would be good and to really get people to come and actually enjoy the efforts it took to make the casino how it is, then having a 25% loss on cash is better than losing 65% of the cash.

**Comment by QuintenM911** [comment]
you keep implying that the casino will always take 65% from you in your statements thinking of it as some permanent tax. when in reality its not a tax its that the casino has no money. when you go in and spin the wheel and take those chips and bet and make money from a casino you have added no money to the casino but have taken chips (money) out. no one right now is going to the casino and selling the chips back or buying large amounts of chips from the casino to gamble so the casino have no money to pay you out. The reason for 35% is so you dont turn 10k chips wheel spin into 290k chip on horses and then cash out and add 290k into the economy from thin air. If the casino has 290k in the account you would get all 100% back but the casino is broke so their is a hold till it is back in the black.

**Comment by ItsTrihawk** [comment]
Idk how accurate this thread has been because I've been paid out in full multiple times lately while I see everyone talk about this "tax" I believe it's something that only someone with developer powers would be able to answer correctly and to give us a true explanation on how it works!

**Comment by BigTony7** [comment]
the reason why the casino is broke is because of the big tax it has, people dont want to go because they know the casino is still broke so no one bothers to buy chips to put money in. im not saying to do a 1 to 1 return on chips, even then the economy wont even be affected, there isnt much to hurt if the only thing you can buy are cars and some pistols, even in crim life there isnt really much to gain from clean cash, rasiing the tax to 75 or even 70% would give people a reason to head to the casino and buy chips. like i said before, you buy 5k worth of chips and barely win 10k and you want to cash out but since the casino is broke you get back 3,500 which you lose money. if the tax doesnt change most people would just do the wheel spin, check if you can cash out 1 to 1 if not they will leave the casino.
